app/views.py

```diff
@@ -600,8 +604,8 @@ def remove_food_resource_type():
 	return jsonify(success="success")
 
 
-@app.route('/_search_query', methods=['GET', 'POST'])
-def save_search_query():
+@app.route('/_search_query_zipcode', methods=['GET', 'POST'])
+def save_search_query_zipcode():
```

Comment:
I was going to have a different search query given a zipcode or address, but then i realized that there was a better way of doing that, and i forgot to change the search_query back to the previous form.